react/components/FloatingActionBar/index.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Button from '../Button'
+
+import './action-bar.global.css'
+
+const FloatingActionBar = ({ onSave, onCancel, cancelLabel, saveLabel }) => (
+  <div className="styleguide__floating-action-bar shadow-active w-100 bg-base tr pv5 pr7-ns absolute fixed bottom-0 left-0 z-2">
+    <span className="mr5">
+      <Button variation="secondary" onClick={onCancel}>
```

Comment:
Shouldn't it be `tertiary` according to the issue?

Reply:
Yes, thanks! Depending how your monitor is moved a secondary button is the same color as a tertiary